Push to nylas

Eagles2F opened the pull request:
Mess around with this code repo. And successfully make it work and deployed on heroku to play with. 
1. token parsing is buggy. I fix it.
2. some packages are updated. I manage to figure a combination to pass the building.
3. the set-up flow is pretty much impossible for me to do it on Digital Ocean or my own mac without Heroku. I would recommend add documentation about setting up this code on heroku free VPS which will ease all the pain in the environment setting.

Eagles2F committed:
update coffemiddle version number

Eagles2F committed:
update api name

Eagles2F committed:
update dependencies

Eagles2F committed:
update redirect uri

Eagles2F committed:
modify this

Eagles2F committed:
fix access_token parsing error

Eagles2F committed:
remove app secret

bengotow commented:
Hey Eagles2F—thanks for submitting a pull request. Your fix to the token parsing is great and we definitely needed the changes to package.json (the install fails for me as well without your changes.) I'm not sure we want to remove `public/js/angular-inbox.js` from the gitignore file, though. Technically, that file should be built in the post-install phase and moved there. The actual source for it is in the `inbox.js` repository.

Could you put that line back in the gitignore? Other than that this looks great. I totally agree that the setup is convoluted. I'll see if there's something we can do to make this simpler. We could potentially merge the inbox.js repository and this one together, and I think it'd make everyone's lives easier.

Last but not least—could you sign our contributor license agreement here? https://nylas.com/cla.html

Thanks!

Eagles2F committed:
add inbox.js back to git ignore

Eagles2F commented:
Hi Ben,
 Nice to hear from you. I've already signed the same agreement in the
nylas/sync-engine project. You could check it out in your database:
Eagles2F

 I also updated the .gitignore. It's a mistake from my part. I shouldn't
delete it.

On Sun, Jul 26, 2015 at 10:31 AM, Ben Gotow <notifications@github.com>
wrote:

> Hey Eagles2F—thanks for submitting a pull request. Your fix to the token
> parsing is great and we definitely needed the changes to package.json (the
> install fails for me as well without your changes.) I'm not sure we want to
> remove public/js/angular-inbox.js from the gitignore file, though.
> Technically, that file should be built in the post-install phase and moved
> there. The actual source for it is in the inbox.js repository.
>
> Could you put that line back in the gitignore? Other than that this looks
> great. I totally agree that the setup is convoluted. I'll see if there's
> something we can do to make this simpler. We could potentially merge the
> inbox.js repository and this one together, and I think it'd make everyone's
> lives easier.
>
> Last but not least—could you sign our contributor license agreement here?
> https://nylas.com/cla.html
>
> Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nylas/inbox-scaffold-html5/pull/10#issuecomment-125019991>
> .
>



-- 
Evan Lee(Yifan Li)
Master of Information Technology-Mobility Candidate
CMU-INI-MS26
412-418-6588
----------------------------------------------------------
祝  健康

李一帆

thueeliyifan@gmail.com

Eagles2F committed:
organize indent

bengotow commented:
Ahh great! Sorry, I didn't notice you'd also contributed to the sync-engine project. Looks great!
